test/unit/vl-select-location.test.html

```diff
@@ -177,41 +177,24 @@
         ]);
       });
 
-      test('er zijn default placeholders voor het zoek adres select element', () => {
+      test('er zijn default placeholders voor het zoek adres select element', async () => {
         const select = fixture('vl-select-location-fixture');
         assert.equal(select._placeholderElement.innerText, 'Lokaliseer adres');
-        assert.equal(select.getTranslation('select.search_placeholder_value'), 'Zoeken op adres of coördinaat');
-        assert.equal(select.getAttribute('data-vl-select-search-empty-text'), 'Geen adres gevonden');
-        assert.equal(select.getTranslation('select.no_more_options'), 'Geen adres gevonden');
+        assert.equal(select.DEFAULT_SEARCH_PLACEHOLDER, 'Zoeken op adres of coördinaat');
+        assert.equal(select.DEFAULT_SEARCH_NO_RESULT, 'Geen adres gevonden');
+        assert.equal(await getNoMoreOptionsText(select), 'Geen adres gevonden');
```

Comment:
Waarom wordt de no-more-option-text hier rechtstreeks gecontroleerd en controleer je alleen de getters bij de andere? Ik vermoed dat ook hier de `DEFAULT_NO_MORE_OPTIONS` getter getest moet worden.

Reply:
Bijgevolg kan ook `getNoMoreOptionsText`, `sendBackspace` en `sendKey` verwijderd worden?

Reply:
Ik wilde hier vooral 'zien' dat die tekst correct was. Aangezien er hier aan een option was met een default placeholder, kon je die tekst niet zomaar zien en is de getNoMoreOptionsText functie iets anders dan in de base class.
Maar, als je er hier inderdaad ook van uit gaat, dat het in de sub class ook  goed getoond wordt (zoals we nu eigenlijk ook bij de andere teksten hebben gedaan, kunnen we hier evengoed ook gewoon tegen die default getter testen.
Kies maar. Als je wil dat ik het aanpas, pas ik het aan...

Reply:
Op dit moment wordt op `vl-select-location` code niveau alleen de getter overschreven dus zou ik het hier beperken tot het testen van de getter. Op niveau van vl-select wordt de functionaliteit al op unit niveau getest. Ik vermoed dat het zetten van een placeholder geen invloed heeft op het tonen van de tekst wanneer er geen opties meer beschikbaar zijn. Ik zou daardoor opteren om de code te verwijderen.